django/contrib/auth/mixins.py

```diff
@@ -73,6 +74,15 @@ def dispatch(self, request, *args, **kwargs):
         return super().dispatch(request, *args, **kwargs)
 
 
+class LoginNotRequiredMixin:
+    """
+    Mixin for CBV that marks that the view is accessible by unauthenticated
+    users.
+    """
```

Comment:
Follow the pattern of the other mixins and use a simple sentence like so:

```suggestion
    """
    Mark the view as accessible by unauthenticated users.
    """
```